jetty-deploy/src/main/java/org/eclipse/jetty/deploy/providers/ScanningAppProvider.java

```diff
@@ -321,4 +323,15 @@ public void setScanInterval(int scanInterval)
     {
         _scanInterval = scanInterval;
     }
+
+    /* ------------------------------------------------------------ */
```

Comment:
Remove this line!

Reply:
Just this one? or all of them in the entire file?